docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -632,7 +680,9 @@ Builtin variables are available under the `builtin.` prefix. Some examples:
               the template and validating if the result is a valid YAML or JSON value.
     - We will only implement syntax validation. A semantic validation is not possible as the patches will be applied 
       to provider templates. **It’s the responsibility of the ClusterClass author to ensure the patches are semantically valid**.
-
+  - `MachineHealthCheck`
+    - If defined at `ClusterClass.spec.machineHealthCheck` must not be defined in `ClusterClass.spec.workers.machineDeployments[i].machineHealthCheck` or `ClusterClass.spec.controlPlane.machineHealthCheck` 
```

Comment:
```suggestion
```
Given that we have MHC only at the leaf level

Reply:
Should be updated now.